assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -110,4 +110,54 @@ describe('SaptuneDetails', () => {
       staging.solutions_ids
     );
   });
+
+  it.each([
+    { solution: null },
```

Comment:
why all of this strange things?
Except the `null` option the other 2 are not real cases. They won't happen, so we don't need to test them.

Reply:
Fair enough, the other cases are irrelevant, i just thought some extra error testing won't harm :D